### PR TITLE
range generator was trying to check for extensions

### LINKED ISSUE #1315: can't change isoline interval in overlay
If I overlay isolines over an isofill plot and then try to change the contour interval of the isolines, I get an error:

Traceback (most recent call last):
  File "/Users/potter/uvcdat/build-uvcdat/install/vistrails/vistrails/gui/uvcdat/graphicsMethodsWidgets.py", line 503, in generateRanges
    if str(self.ext1ButtonGroup.buttonGroup.button(self.ext1ButtonGroup.buttonGroup.checkedId()).text()).lower()[0]=="y":
AttributeError: 'QIsolineEditor' object has no attribute 'ext1ButtonGroup'

### PR DESCRIPTION
**Pull request by doutriaux1**
but in case of isoline there are no extensions
also in case of isolines we needed to generate one extra color
fix https://github.com/UV-CDAT/uvcdat/issues/1315

### CODE REVIEW

**Comment by doutriaux1** [comment]
@vvpalav @potter2 @remram44 this should fix https://github.com/UV-CDAT/uvcdat/issues/1315

**Comment by doutriaux1** [comment]
@remram44 @vvpalav @hgohil2805 this is holding release 2.2.0 
@williams13 can we do without it?

**Comment by doutriaux1** [comment]
@aashish24 in the interest of time do you mind reviewing this? Thanks!

**Comment by doutriaux1** [comment]
@chaosphere2112 @dlonie same please

File: vistrails/gui/uvcdat/graphicsMethodsWidgets.py [hunk 1]
```diff
@@ -500,17 +500,20 @@ def generateRanges(self):
...
+        else:
+            colors = vcs.getcolors(values+[values[-1],])
 
-        if str(self.ext2ButtonGroup.buttonGroup.button(self.ext1ButtonGroup.buttonGroup.checkedId()).text()).lower()[0]=="y":
+        if hasattr(self,"ext2ButtonGroup") and str(self.ext2ButtonGroup.buttonGroup.button(self.ext1ButtonGroup.buttonGroup.checkedId()).text()).lower()[0]=="y":
           ## Ok we want a right extension
           if abs(values[-1])<1.e20:
             values.append(1.e20)
 
-        colors = vcs.getcolors(values)
```
Comment (aashish24) [inline vistrails/gui/uvcdat/graphicsMethodsWidgets.py:513]: assuming that this code is not required, I think this branch looks good to me.

Comment (doutriaux1) [inline vistrails/gui/uvcdat/graphicsMethodsWidgets.py:513]: it has been moved up @aashish24 line 508 or 510 now.

**Pull request merged**